installer/create_installer.sh

```diff
@@ -2,10 +2,11 @@
 
 cd "$(dirname "$0")"
 
-VERSION=$(grep ^VERSION ../setup.py | awk '{ print $3 }' | sed "s/'//g" )
+VERSION=$(grep ^__version__ ../ldm/invoke/_version.py | awk -F= '{ print $2 }' | sed "s/'//g" | sed -r 's/\s+//g' )
```

Comment:
you could use

`VERSION=$(cd .. && python -c "from ldm import invoke; print(invoke.__version__)")`

Reply:
Change implemented.